src/containers/ruby-tab/sound-completer.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Define Ruby code completer for Motion Blocks
+ */
+class SoundCompleter {
+    getCompletions (editor, session, pos, prefix, callback) {
+        console.log("pos: ", pos);
+        const words = [
+            {
+                value: 'play_until_done("ニャー")',
+                meta: '終わるまでニャーの音を鳴らす' 
+            },
+            {
+                value: 'play("ニャー")',
+                meta: 'ニャーの音を鳴らす'
+            },
+            {
+                value: 'stop_all_sounds',
+                meta: 'すべての音を止める'
+            },
+            {
+                value: 'change_sound_effect_by("PITCH",10)',
+                meta: 'ピッチの効果を10ずつ変える'
+            },
+            {
+                value: 'set_sound_effect("PITCH", 100)',
+                meta: 'ピッチの効果を100にする'
+            },
+            {
+                value: 'clear_sound_effects',
+                meta: '音の効果をなくす'
+            },
+            {
+                value: 'self.volume += -10',
+                meta: '音量を-10ずつ変える'
+            },
+            {
+                value: 'self.volume =100',
+                meta: '音量を100%にする'
+            },
+            {
+                value: 'volume',
+                meta: '音量'
+            }
+        ];
+        let completions = [];
+        words.forEach(function(w) {
+            completions.push(w);
+        });
```

Comment:
プログラムの書き方が間違っていましたので、あべくんが提案してくれたものにします。

```suggestion
        words.forEach(w => completions.push(w));
```

関連するエラー
```
  46:23  error  Unexpected function expression                          prefer-arrow-callback
  46:31  error  Missing space before function parentheses               space-before-function-paren
```